### PR TITLE
Mime type

### PR DESCRIPTION
**Pull request by tarzzz**

### CODE REVIEW

**Comment by tarzzz** [comment]
Good to be merged

File: pydy_viz/camera.py [hunk 1]
```diff
@@ -102,8 +102,13 @@ def __init__(self, *args, **kwargs):
...
             #reference frame
-            self._reference_frame = args[i]
-            i += 1
+            try:
+                dcm = args[i]._dcm_dict
+                self._reference_frame = args[i]
+                i += 1
+            except AttributeError:
+                raise TypeError(''' A ReferenceFrame is to be supplied
+                                   before a Particle/Point. ''')
```
Comment (moorepants) [inline pydy_viz/camera.py:111]: What does this have to do with mime type?

**Comment by moorepants** [comment]
Does this fix things for running in firefox?

**Comment by tarzzz** [comment]
yes it totally does ..

**Comment by moorepants** [comment]
If you've tested this for Firefox and it all works on your end, then you can merge it.

Does this fix issue #39 too?

**Comment by tarzzz** [comment]
Yes it does .. I will merge it in a few minutes
